browser/src/control/Control.JSDialogBuilder.js

```diff
@@ -2491,14 +2474,11 @@ L.Control.JSDialogBuilder = L.Control.extend({
 
 			L.DomUtil.addClass(div, 'uno' + id);
 
-			if (isRealUnoCommand)
-				id = builder._makeIdUnique(id);
-
 			div.id = id;
 			data.id = id; // change in input data for postprocess
 
 			var icon = data.icon ? data.icon : builder._createIconURL(data.command);
-			var buttonId = id + 'img';
+			var buttonId = id + '-button';
```

Comment:
isn't that 'img' used in some branding CSS or in CSS in general?

Reply:
I will also update those parts i think. This is draft for now.